src/dotnet/devcontainer-feature.json

```diff
@@ -9,8 +9,8 @@
             "type": "string",
             "proposals": [
                 "latest",
+                "7",
```

Comment:
For `latest` looks like we have hardcoded to `6`. Should we update that as well? 

https://github.com/devcontainers/features/blob/285097064cf58bd228e1a231421ccdf9feb894f4/src/dotnet/install.sh#L213

If we decide to do so, then can we bump the minor version instead? 😄

Reply:
Ugh - yeah that needs to be updated.  We also shouldn't really hard code this - but that's better for a different PR.

Reply:
Looks like the tests are also going to return false positives since the `| grep ...` would apply to the output of the test command. Will fix that.